src/components/__tests__/ProtectedRoute.test.js

```diff
@@ -60,12 +67,71 @@ describe('The ProtectedRoute component', () => {
     );
 
     expect(wrapper.find(TestComponent).exists()).toBe(true);
+    expect(wrapper.find(Snackbar).prop('open')).toBe(false);
     expect(wrapper.find(Route).exists()).toBe(true);
     expect(wrapper.find(Route).prop('path')).toBe('/');
 
     MockDate.reset();
   });
 
+  test('renders message and component when authenticated', () => {
```

Comment:
I like that it's in ProtectedRoute so that we only have to test the actual rendering once. Everywhere else, we can just test that the action was dispatched.